provision.py

```diff
@@ -110,6 +110,21 @@
 
 LOUD = dict(_out=sys.stdout, _err=sys.stderr)
 
+def setup_node_modules():
+    output = subprocess.check_output(['sha1sum', 'package.json'])
+    sha1sum = output.split()[0]
+    success_stamp = os.path.join(os.environ['HOME'], 'success-stamp', sha1sum)
+    if not os.path.exists(success_stamp):
+        print("Deleting cached version")
+        run(["rm", "-rf", "node_modules"])
+        print("Installing node modules")
+        run(["npm", "install"])
+        run(["rm", "-rf", success_stamp])
+        run(["mkdir", "-p", success_stamp])
```

Comment:
The second `rm -rf` is no longer needed, since now the `success_stamp`s are inside `node_modules`...